linode/manager
# M3-5947: Switch to Akamai Billing Information and Tax IDs

## Description 📝

- Makes Cloud Manager use Akamai information and Tax IDs for PDF invoices starting `2022-12-15`
- Shows Akamai's remit address dynamically
  - Shows Linode address of any invoice before `2022-12-15`
  - Shows Akamai  US address for any invoice on or after `2022-12-15`
  - Shows Akamai international address for any invoice on or after `2022-12-15` for accounts with country that is not US or Canada
- Makes the `date` value for a tax in LD optional so taxes can have no set start date

> **Note**: Tax IDs are stored in a feature flag

## Preview 📷

## Before
![Screen Shot 2022-12-09 at 11 34 03 AM](https://user-images.githubusercontent.com/115251059/206748603-6c71f69d-5747-4fd9-8655-40dcb92000fe.jpg)

## After
![Screen Shot 2022-12-09 at 11 34 09 AM](https://user-images.githubusercontent.com/115251059/206748611-9b6c10ae-fdb4-49a8-a657-54ceef35a793.jpg)


## How to test 🧪

> **Note:** You can enable the MSW and view your invoices by going to `http://localhost:3000/account/billing` to mock future invoices

- Go to `/account/billing` and view a PDF invoice
- If an invoice date is before `2022-12-15`, the Linode `Remit To` address should be shown in the invoice
  - The Tax IDs shown in that invoice should be Linode's Tax IDs
- If an invoice date is after `2022-12-15`, the Akamai `Remit To` address should be shown in the invoice
  - If the customer's address is in the US or Canada, you should see Akamai's US address, otherwise you should see Akamai's International address 
  - The Tax IDs shown in that invoice should be Akamai's Tax IDs
